_api-reference/profile.md

```diff
@@ -987,18 +987,18 @@ The following sections contain definitions of all modified or added response fie
 
 |Field	|Description	|
 |:---	|:---	|
-|`time_in_nanos`	|The total elapsed time for this query, in nanoseconds. For concurrent segment search, `time_in_nanos` is the total amount of time across all slices (`max(slice_end_time) - min(slice_start_time)`).   |
-|`max_slice_time_in_nanos`	|The maximum amount of time taken by any slice to run a query, in nanoseconds.	|
-|`min_slice_time_in_nanos`	|The minimum amount of time taken by any slice to run a query, in nanoseconds.	|
-|`avg_slice_time_in_nanos`	|The average amount of time taken by any slice to run a query, in nanoseconds.	|
-|`breakdown.<method>`	|For concurrent segment search, this is the total elapsed time across all slices (`max(slice_end_time) - min(slice_start_time)`). For example, for the `build_scorer` method, it is the total time spent constructing `Scorer` object across all slices.     |
-|`breakdown.max_<method>`	|The maximum amount of time taken by any slice to run a query method. `create_weight` method does not have profiled max time stats because they are executed at the query-level rather than the slice-level.	|
-|`breakdown.min_<method>`	|The minimum amount of time taken by any slice to run a query method. `create_weight` method does not have profiled min time stats because they are executed at the query-level rather than the slice-level.	|
-|`breakdown.avg_<method>`	|The average amount of time taken by any slice to run a query method. `create_weight` method does not have profiled avg time stats because they are executed at the query-level rather than the slice-level.	|
-|`breakdown.<method>_count`	|For concurrent segment search, this field contains the total number of invocations of a `<method>` obtained by adding the number of method invocations for all segments.       |
-|`breakdown.max_<method>_count`	|The maximum number of invocations of a `<method>` on any slice. `create_weight` method does not have profiled avg count stats because they are executed at the query-level rather than the slice-level.       |
-|`breakdown.min_<method>_count`	|The minimum number of invocations of a `<method>` on any slice. `create_weight` method does not have profiled avg count stats because they are executed at the query-level rather than the slice-level.       |
-|`breakdown.avg_<method>_count`	|The average number of invocations of a `<method>` on any slice. `create_weight` method does not have profiled avg count stats because they are executed at the query-level rather than the slice-level.       |
+|`time_in_nanos`	| The total elapsed time for this query, in nanoseconds. For concurrent segment search, `time_in_nanos` is the total amount of time across all slices (`max(slice_end_time) - min(slice_start_time)`).   |
```

Comment:
`slice_end_time` and `slice_start_time` is not exposed/known to users. We should instead replace it with wording something like: _For concurrent segment search, `time_in_nanos` is the total time spent across all the slices i.e. difference between end time of last completed slice execution and start time of first slice execution._

Reply:
Hi @sohami, thanks for the review. It's been addressed now.